tests/pom.xml

```diff
@@ -145,6 +152,10 @@
                     <runOrder>alphabetical</runOrder>
                     <failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
                     <parallelTestsTimeoutInSeconds>360</parallelTestsTimeoutInSeconds>
+		    <environmentVariables>
```

Comment:
Shouldnt be necessary because it is activated via `KC_COMMUNITY_DATASTORE_CASSANDRA_ENABLED=true` env variable